### PR TITLE
Feat:Refactored and Validation added

### PR DESCRIPTION
**Pull request by varun2948**

### CODE REVIEW

File: src/frontend/main/src/components/createproject/UploadArea.tsx
Comment (Mudi-business) [inline src/frontend/main/src/components/createproject/UploadArea.tsx:1]: I think you forgot to import the INPUT component in the CoreModules.

**Review by Mudi-business — COMMENTED** [review]
I think everything else looks good @varun2948

**Comment by varun2948** [comment]
@Mudi-business ohh thanks for pointing it out mudi , i have forgotten while testing it, i will send another commit removing that persist.

File: src/frontend/main/src/components/createproject/UploadArea.tsx
Comment (varun2948) [reply src/frontend/main/src/components/createproject/UploadArea.tsx:1]: Ohh yeah, i sud do that aswell

File: src/frontend/main/src/store/Store.js
[the remark this answers is not in the document]
Comment (Mudi-business) [reply src/frontend/main/src/store/Store.js:1]: > @Mudi-business ohh thanks for pointing it out mudi , i have forgotten while testing it, i will send another commit removing that persist.

Thank you , for your good work

**Pull request merged**